pgx-examples/spi_srf/spi_srf.control

```diff
@@ -0,0 +1,6 @@
+comment = 'spi_srf:  Created by pgx'
+default_version = '@CARGO_VERSION@'
+module_pathname = '$libdir/spi_srf'
+relocatable = false
+superuser = false
+schema = 'spi_srf'
```

Comment:
Needs a newline at the end of this file. (Sorry, pedantic)

Reply:
Oh, Okay, I just added the extra line. Thanks!